crawl-ref/source/dat/strings/de/hiscores.txt

```diff
@@ -671,25 +671,25 @@ Zu Tode geknabbert durch Software-Bugs
 %%%%
 # %s could be a monster or a thing
 ... while paralysed by %s
-... während der Lähmung von {dat}%s
+... während einer Lähmung durch {dat}%s
 %%%%
 ... while paralysed
-... während er gelähmt ist
+... während einer Lähmung
 %%%%
 # %s is always a monster
 ... while petrified by %s
-... während der Versteinerung von {dat}%s
+... während einer Versteinerung durch {dat}%s
```

Comment:
Shouldn't this be "von"?
For example, "... während einer Versteinerung von einem Basilisk"

Reply:
With "von" it sounds like you were petrifying the basilisk. The "durch" makes it clear it was the other way around. Thinking about it, we probably need to change {dat} into {acc}? With "durch" it should be "einen" not "einem":
"... während einer Versteinerung durch einen Basilisk"